doc/api/deprecations.md

```diff
@@ -0,0 +1,574 @@
+[`os.networkInterfaces`][] property instead.
+
+<a id="DEP0024"></a>
+### DEP0024: REPLServer.prototype.convertToContext()
+
+Type: Runtime
+
+The `REPLServer.prototype.convertToContext()` API is deprecated and should
+not be used.
+
+<a id="DEP0025"></a>
+### DEP0025: require('sys')
+
+Type: Runtime
+
+The `sys` module is deprecated. Please use the [`util`][] module instead.
+
+<a id="DEP0026"></a>
+### DEP0026: util.print()
+
+Type: Runtime
+
+The [`util.print()`][] API is deprecated. Please use [`console.log()`][]
+instead.
+
+<a id="DEP0027"></a>
+### DEP0027: util.puts()
+
+Type: Runtime
+
+The [`util.puts()`][] API is deprecated. Please use [`console.log()`][] instead.
+
+<a id="DEP0028"></a>
+### DEP0028: util.debug()
+
+Type: Runtime
+
+The [`util.debug()`][] API is deprecated. Please use [`console.error()`][]
+instead.
+
+<a id="DEP0029"></a>
+### DEP0029: util.error()
+
+Type: Runtime
+
+The [`util.error()`][] API is deprecated. Please use [`console.error()`][]
+instead.
+
+<a id="DEP0030"></a>
+### DEP0030: SlowBuffer
+
+Type: Documentation-only
+
+The [`SlowBuffer`][] class has been deprecated. Please use
+[`Buffer.allocUnsafeSlow(size)`][] instead.
+
+<a id="DEP0031"></a>
+### DEP0031: ecdh.setPublicKey()
+
+Type: Documentation-only
+
+The [`ecdh.setPublicKey()`][] method is now deprecated as its inclusion in the
+API is not useful.
+
+<a id="DEP0032"></a>
+### DEP0032: domain module
+
+Type: Documentation-only
+
+The [`domain`][] module is deprecated and should not be used.
+
+<a id="DEP0033"></a>
+### DEP0033: EventEmitter.listenerCount()
+
+Type: Documentation-only
+
+The [`EventEmitter.listenerCount(emitter, eventName)`][] API has been
+deprecated. Please use [`emitter.listenerCount(eventName)`][] instead.
+
+<a id="DEP0034"></a>
+### DEP0034: fs.exists(path, callback)
+
+Type: Documentation-only
+
+The [`fs.exists(path, callback)`][] API has been deprecated. Please use
+[`fs.stat()`][] or [`fs.access()`][] instead.
+
+<a id="DEP0035"></a>
+### DEP0035: fs.lchmod(path, mode, callback)
+
+Type: Documentation-only
+
+The [`fs.lchmod(path, mode, callback)`][] API has been deprecated.
+
+<a id="DEP0036"></a>
+### DEP0036: fs.lchmodSync(path, mode)
+
+Type: Documentation-only
+
+The [`fs.lchmodSync(path, mode)`][] API has been deprecated.
+
+<a id="DEP0037"></a>
+### DEP0037: fs.lchown(path, uid, gid, callback)
+
+Type: Documentation-only
+
+The [`fs.lchown(path, uid, gid, callback)`][] API has been deprecated.
+
+<a id="DEP0038"></a>
+### DEP0038: fs.lchownSync(path, uid, gid)
+
+Type: Documentation-only
+
+The [`fs.lchownSync(path, uid, gid)`][] API has been deprecated.
+
+<a id="DEP0039"></a>
+### DEP0039: require.extensions
+
+Type: Documentation-only
+
+The [`require.extensions`][] property has been deprecated.
+
+<a id="DEP0040"></a>
+### DEP0040: punycode module
+
+Type: Documentation-only
+
+The [`punycode`][] module has been deprecated. Please use a userland alternative
+instead.
```

Comment:
why not ", it is now available from npmjs.org, https://www.npmjs.com/package/punycode"